src/main/scala/nl/knaw/dans/easy/stage/fileitem/EasyFileMetadata.scala

```diff
@@ -15,13 +15,18 @@
  */
 package nl.knaw.dans.easy.stage.fileitem
 
+import java.io.File
+
 import scala.util.Try
 
 object EasyFileMetadata {
   def apply(s: FileItemSettings): Try[String] = Try {
+      val parentPath = s.pathInDataset.get.getParentFile
```

Comment:
Thanks @GeorgiKhomeriki, good to hear from you. I'm starting to think I should have finished reading Real World Haskell after all :-/